### ISSUE TITLE
Fix Travis setup

### ISSUE THREAD

**Issue by DanielMSchmidt** [lelandrichardson/react-native-mock#25]
Hi there,

Thanks again for this great lib. I was just curious if you would mind fixing your travis setup? A simple `travis.yml` with `language: node_js` should work as it automatically runs `npm install` and `npm test`.

Have a nice day!

**Comment by bdougie** [comment]
@DanielMSchmidt I am not familiar with Travis and have not used it for a project, but just by took a look at the react-native travis.yml.

Is what your asking for something that looks like this?

```
//.travis.yml

language: node_js
node_js:
  - "5"
  - "5.1"
  - "4"
  - "4.2"
  - "4.1"
```

**Comment by DanielMSchmidt** [comment]
@bdougie Yeah, that would be okay

**Comment by DanielMSchmidt** [comment]
@bdougie Yeah, that would be okay

**Comment by RealOrangeOne** [comment]
Issue was fixed in #29
